src/frontend/js/modules/provider.js

```diff
@@ -17,6 +17,9 @@ const _ = require('lodash');
       }
 
       Providers.prototype = {
+        getNumberProviders: function() {
```

Comment:
```suggestion
        getNumberOfProviders: function() {
```

Reply:
I thought carefully before choosing the name :trollface: The word "Of" does not provide any more meaningful explanation to the function purpose, so I discarded it

Reply:
Grammatically speaking, `getNumberOfProviders` is the correct one to use here. `getNumberProviders` means getting the providers, not a number